src/main/openapi/ds-image-openapi_v1.yaml

```diff
@@ -266,6 +266,10 @@ paths:
             Define a region of interest starting at relative coordinates x,y with width w and height h.
             
             For use with CVT requests. All values should be ratios in the range 0 – 1.0
+            
+            When using the RGN parameter, please note that the WID parameter is **NOT** automatically adjusted. 
+            If WID is not adjusted accordingly you will receive an upscale version of the region requested. 
+            If you don't know the original width of the image, this can be obtained through the ```/IIIF/{identifier}/info.{format}```-endpoint.  
```

Comment:
1. Could you mention that `WID` defaults to the original image width in pixels? As computer nerds we are used to think in pixels, but I know that at least some design oriented people are more oriented towards physical dimensions and resolutions
2. The "If you don't know"-modifier sounds a bit condescending. Maybe just remove it and simply state how the width can be obtained.